app/components/ui/Typography/Title.tsx

```diff
@@ -23,14 +23,16 @@ export const Title3 = ({ children, className = "" }: TitleProps) => {
 };
 
 export function TitleComponent({ title, titleSize, bold }: TitleProps) {
+  const boldClass = bold ? "md:font-bold" : "md:font-normal";
```

Comment:
Small refactor to move this logic into the TitleComponent itself. So we don't need to compute this in every place we use it.